configure.ac

```diff
@@ -4,7 +4,10 @@ AC_INIT([collectd],[m4_esyscmd(./version-gen.sh)])
 AC_CONFIG_SRCDIR(src/target_set.c)
 AC_CONFIG_HEADERS(src/config.h)
 AC_CONFIG_AUX_DIR([libltdl/config])
-AC_CONFIG_SUBDIRS([src/libmongoc])
+if test "x$with_libmongoc" = "xown"
+then
+	AC_CONFIG_SUBDIRS([src/libmongoc])
```

Comment:
Can you just put this around line 3100 in the existing conditional?

Reply:
All set, I moved this to the existing check for `with_libmongoc == own`